### DISCUSSION TITLE
web-frameworks-benchmark

### DISCUSSION THREAD

**Discussion by josejachuf** [salvo-rs/salvo#362] [General]
What happened to this?
Until a few days ago it was located in the green zone.

![imagen](https://github.com/salvo-rs/salvo/assets/5981345/fc5baac6-0ac4-4470-b3be-60455ac6ac97)

https://web-frameworks-benchmark.netlify.app/result?l=rust

**Comment by chrislearn** [comment]
I believe this was caused by a bug and has already been fixed.
Let's take a look at the results of the next round of testing.
